File: spam.js
export default function handler(req, res) {
  const { word = "SPAM", count = 1 } = req.query;

  let num = parseInt(count);
  if (isNaN(num) || num < 1) num = 1;
  if (num > 50) num = 50;

  const lines = Array(num).fill(word).join("\n");

  res.setHeader("Content-Type", "text/plain; charset=utf-8");
  res.send(lines);
}

File: spam.py
from flask import Request, Response

def handler(request: Request) -> Response:
    word = request.args.get("word", "SPAM")
    count = int(request.args.get("count", 1))
    if count > 50:
        count = 50
    result = "\n".join([word] * count)
    return Response(result, mimetype="text/plain")
